src/server/svr_jobfunc.c

```diff
@@ -3810,6 +3811,8 @@ eval_resvState(resc_resv *presv, enum resvState_discrim s, int relVal,
 				else
 					*psub = RESV_RUNNING;
 				*pstate = RESV_RUNNING;
+				if (presv->ri_qs.ri_tactive < presv->ri_wattr[RESV_ATR_start].at_val.at_long)
```

Comment:
@riyazhakki thanks for making this change. Can you please add comment stating why we are doing this.